traffic_portal/app/src/common/service/utils/UrlUtils.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var UrlUtils = function() {
+    this.getUrlQueryParams = function(url) {
+        var vars = {};
+        var hashes = url.split('?')[1];
+        var hash = hashes != null ? hashes.split('&') : '';
+
+        for (var i = 0; i < hash.length; i++) {
+            var params=hash[i].split('=');
+            vars[params[0]] = params[1];
+        }
+        return vars;
+    };
+};
+UrlUtils.$inject = [];
+module.exports = UrlUtils;
```

Comment:
newlines at the end of files for POSIX-compliance, please

Reply:
deleted this utils file and moved it all to just use the URL object
done